src/main/resources/templates/fragments/dashboard_navigation.html

```diff
@@ -41,13 +41,15 @@
 			<ul class="nav navbar-nav navbar-left login-navbar-left">
 				<li><a href="/teams" th:href="@{/teams}"><span th:text="#{nav.teams}">Teams</span></a></li>
 				<li><a href="/experiments" th:href="@{/experiments}"><span th:text="#{nav.exp}">Experiment</span></a></li>
-				<li><a href="/data" th:href="@{/data}">Data</a></li>
+				<!--<li><a href="/data" th:href="@{/data}">Data</a></li>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)